### PR TITLE
Add http /readyz sidecar to kube-apiserver pod

### PR DESCRIPTION
**Pull request by sttts**
Needed for GCP load balancers. They don't support https.

### CODE REVIEW

**Comment by sttts** [comment]
@abhinavdahiya have to test this first. Just wrote it down blindly.

File: bindata/v4.1.0/kube-apiserver/pod.yaml [hunk 1]
```diff
@@ -99,6 +99,21 @@ spec:
       name: resource-dir
     - mountPath: /etc/kubernetes/static-pod-certs
       name: cert-dir
+  - name: kube-apiserver-insecure-readyz-REVISION
+    image: ${OPERATOR_IMAGE}
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command: ["cluster-kube-apiserver-operator", "insecure-readyz"]
+    args:
+    - --insecure-port=6080
```
Comment (abhinavdahiya) [inline bindata/v4.1.0/kube-apiserver/pod.yaml]: Any reason why we picked 6080,
this is quite separate/far from 6443, something like 6440 to give room and still close enough...

Not important, feel free to ignore we can program lbs to use anything :)

Comment (sttts) [reply bindata/v4.1.0/kube-apiserver/pod.yaml]: 6000 + 443 vs. +80

**Comment by abhinavdahiya** [comment]
https://github.com/openshift/library-go/pull/478 was merged.

**Comment by sttts** [comment]
rebased

**Comment by sttts** [comment]
@abhinavdahiya now this needs to be wired into the bootkube script in the installer.

**Pull request merged**